pkg/provisioner/resourcehandlers.go

```diff
@@ -53,6 +53,10 @@ const (
 	objectBucketNameFormat = "obc-%s-%s"
 )
 
+func configMapNameForClaim(obc *v1alpha1.ObjectBucketClaim) string {
```

Comment:
1) why is this not in helpers? 2) why isn't there a parallel func for secret names? 3) the func name pattern is used through the lib but in all case but here the func returns is a pointer to the resource and an error. Therfore, I think a different func name is beneficial, eg. `generateConfigMapName` or `makeCMName` etc.

Reply:
1. I'll move that no problem
2. I'm not sure what you are suggesting here. `composeBucketName` returns a string and error, but this function doesn't need to have an error return.

Reply:
Yes regarding `composeBucketName` but my point above is that composeBucketName is not named `bucketNameFromClaim` -- this is the pattern I'm referring to. It's a nit but I noticed it.